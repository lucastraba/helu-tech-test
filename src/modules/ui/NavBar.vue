<template>
  <div class="navigation-bar">
    <b-navbar
      type="dark"
      variant="dark"
    >
      <b-navbar-brand href="#">
        Covid 19 Tracker by Lucas Traba
      </b-navbar-brand>
    </b-navbar>
  </div>
</template>

<script>
export default {
  name: 'NavBar'
};
</script>
