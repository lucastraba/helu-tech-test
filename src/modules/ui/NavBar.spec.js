import { shallowMount } from '@vue/test-utils';
import NavBar from './NavBar';

describe('NavBar Component', () => {
  it('should render the nav bar', () => {
    // Arrange.
    const wrapper = shallowMount(NavBar, {
      stubs: {
        bNavbar: true,
        bNavbarBrand: true
      }
    });
    // Act.
    // Assert.
    expect(wrapper.html()).toMatchSnapshot();
  });
});
