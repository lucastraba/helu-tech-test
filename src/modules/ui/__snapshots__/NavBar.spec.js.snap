// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar Component should render the nav bar 1`] = `
"<div class=\\"navigation-bar\\">
  <bnavbar-stub type=\\"dark\\" variant=\\"dark\\">
    <bnavbarbrand-stub href=\\"#\\">
      Covid 19 Tracker by Lucas Traba
    </bnavbarbrand-stub>
  </bnavbar-stub>
</div>"
`;
